examples/feedforward_nn.rs

```diff
@@ -25,6 +26,10 @@ fn main() {
     let ys = tensor!([[1.], [-1.], [-1.], [1.]], requires_grad = false);
     let mut ypred: Tensor<Ix2> = Tensor::zeros((4, 1));
 
+    let path = "checkpoint1.pkl";
+    save(model.state_dict(), path);
+    println!("{:?}", load(path));
+
```

Comment:
What is the point of this? It seems like this logic belongs in a test since it doesn't add any value to this example file.

Reply:
Yes, you're right. Just added it incase someone wanted to play around with it. Will not include this in the final PR.